pom.xml

```diff
@@ -207,6 +213,14 @@
       </dependency>
 
       <dependency>
+        <groupId>io.syndesis</groupId>
+        <artifactId>integration-runtime-bom</artifactId>
+        <version>${syndesis-integration-runtime.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
```

Comment:
you lost me here. my understanding is that importing a bom affects only `dependencyManagement` nothing that is not used directly is gonna be used.

Reply:
Sorry, didn't mean that they are included automatically.. just that the bom adds into dependency-management third-party libraries such as jackson and others.. and this may lead to selecting a different version of a transitive lib in some module if this repo and the integration-runtime go out of sync..